### ISSUE TITLE
CICD deploy has stopped working

### ISSUE THREAD

**Issue by myfairshare** [aws-amplify/amplify-console#1768]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-console/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-console/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-console/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Describe the bug

Hello,

   I have an amplify app set up with CICD, when I push my changes in vscode to the dev branch my staging environment auto rebuilds and deploys, has been like this for a few months.  Today out of the blue its just stopped working, I have pushed a few times today but all of a sudden nothing is happening, in monitoring on the amplify console I notice some 301 statuses but theres not really any other information, could some please advise what to do, thank you.

### Expected behavior

Build start automatically as per usual

### Reproduction steps

Update my React code
Push with vscode

**Comment by myfairshare** [comment]
Just to follow up, I made a ull request from dev to master and it built as per usual.  But for some reason my staging environment is not showing the last updates I made.  I added a button to a component which is showing in my production master build but not in my dev staging build, but I checked the code in the staging build and the button is there, its like as if the last code push to the dev branch just failed to build in amplify.  Maybe I will just add a new bit of code to my dev branch and try pushing again to see what happens, very strange though, this has never happened before.
